### DISCUSSION TITLE
How does kaspi pay integrate payments using C#,Как kaspi использует c# интегрированные платежи

### DISCUSSION THREAD

**Discussion by StefanieJing** [community/community#73022] [Programming Help]
### Body

Hello, everyone My system wants to use kaspi for payment. The front-end programming language is IOS and Android, and the back-end programming language is C#, but I can't find the official payment documents and related technical support articles of kaspi. Is there any partner who has integrated kaspi payment? Does kaspi officially have code samples on GitHub? Please give me a little help and tips, thank you

Всем привет. Моя система хочет использовать kaspi для оплаты, язык программирования IOS и андроид на переднем конце, а язык программирования на заднем конце — C#, но я не могу найти официальный платежный документ кассапи и связанные с ним статьи технической поддержки. Могу я спросить, есть ли у кассапи официальные примеры кода на GitHub? Пожалуйста, помогите мне с подсказкой

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Khamit** [comment]
навряд ли но может у каспи закрытый код? если кто знает пните тож хочу сделать оплату по номеру каспи
